core/src/main/java/org/hibernate/ogm/hibernatecore/impl/OgmSessionImpl.java

```diff
@@ -218,6 +224,32 @@ public SimpleNaturalIdLoadAccess bySimpleNaturalId(String entityName) {
 		throw new UnsupportedOperationException( "OGM-589 - Natural id look-ups are not yet supported" );
 	}
 
+	@Override
+	public QueryImplementor createQuery(String queryString) {
+		SessionImpl session = (SessionImpl) delegate;
+		QueryImpl queryImplementor = (QueryImpl) session.createQuery( queryString );
+		OgmQueryImplFacade ogmQueryImplFacade = new OgmQueryImplFacade( queryImplementor );
```

Comment:
Maybe you can use 
```java
super.createQuery( queryString)
```
to obtain the instance of **OgmQueryImplFacade**

Reply:
No. It is not enough

Reply:
Sorry. I misspelt the class. I meant only to create **queryImplementor**. the instance that we pass as agrument to **OgmQueryImplFacade**.